Install by default in editable/development mode. (#91)
MichaelAquilina/zsh-autoswitch-virtualenv: README.rst

If you wish to change this to another location, simply set the value of the
environment variable ``AUTOSWITCH_VIRTUAL_ENV_DIR``.

**Customising pip install invocation**

> By default it will invoke pip install in editable (i.e. development) mode. See

Reviewer: Maybe "By default `mkvenv` will invoke pip install in editable....."?

Author: Perhaps it should reference setup i.e. dependencies from requirements are not editable eg.

```suggestion
By default `mkvenv` will install setup.py via pip in editable (i.e. development) mode. See
```

Reviewer: Happy with the suggestion :+1:
